pom.xml

```diff
@@ -41,11 +41,13 @@
 			<groupId>org.springframework.social</groupId>
 			<artifactId>spring-social-core</artifactId>
 			<version>${spring-social.version}</version>
+			<scope>provided</scope>
```

Comment:
We need to be clear on these provided scopes.  Are they provided because another dependency of spring-social-salesforce is including them or is it because of a conflict on the Alfresco side.  If they are coming from the alfresco side we need to exclude them on the connector and not mark them as provided here as this may cause dependency issues with other users of this project.